iree/tools/compilation.bzl

```diff
@@ -57,3 +58,14 @@ def iree_bytecode_module(
             flatten = True,
             **kwargs
         )
+    # Embed the module for use in C.
+    if c_output:
+        c_embed_data(
+            name = "%s_cc" % (name),
```

Comment:
Should have a different name I think?

Reply:
Yes, I notice that when I test the bazel flow. Fixed in the new patch.